components/expenses/ExpenseForm.js

```diff
@@ -570,6 +578,30 @@ const ExpenseFormBody = ({
                     {errors.payoutMethod.data.currency.toString()}
                   </Box>
                 )}
+                <StyledHr flex="1" borderColor="#ffffff" mx={2} />
```

Comment:
Maybe just use `white` ? 

```suggestion
                <StyledHr flex="1" borderColor="white" mx={2} />
```

Reply:
No luck with `white` 😕 .

Reply:
Yes that doesn't work because you need to specify the "type" of white; try this, 😉 

```suggestion
                <StyledHr flex="1" borderColor="white.full" mx={2} />
```

Reference for the `white` styles;

https://github.com/opencollective/opencollective-frontend/blob/9b33639bfad126a681d5561dfce4127456f25371/lib/theme/colors.js#L19-L26

Reply:
Thanks, that works!